lib/jekyll/regenerator.rb

```diff
@@ -28,9 +28,7 @@ def regenerate?(document)
         regenerate_document?(document)
       else
         source_path = document.respond_to?(:path) ? document.path : nil
-        dest_path = if document.respond_to?(:destination)
-                      document.destination(@site.dest)
-                    end
+        dest_path = (document.destination(@site.dest) if document.respond_to?(:destination))
```

Comment:
Why does this need to be in parentheses but line 182 does not?

Reply:
I think this is wrong?

Reply:
...this doesn't 😄